liferay-frontend/liferay-portal
# LPS-126135 - Adding brand colors

Some teams like LOL or liferay.com team need to create brand fragments/content based on branding colors, not in components colors, that's the main reason to add the new colors category "Brand Colors"

About naming: `brand-color-1` `2` `3` `4`
### **Why do we use numbers instead of labeling like "primary" "secondary", etc?**

Branding colors can be pallets with colors with the same importance, so primary, secondary, tertiary.. wouldn't be correct to naming those variables, we finally add 4 number variables to provide flexibility and adaptability.

---

### How to test this PR?

- Go to product menu > style > stylebook > Create a new one
- Go to Color system
- Change the Brand color 4 and see changes in the footer


![image](https://user-images.githubusercontent.com/7963804/127166901-a3299410-0ee5-442f-a2a2-edeb5ed3c931.png)

![image](https://user-images.githubusercontent.com/7963804/127166915-78b7eef5-b2ef-402f-a138-c9df037513e8.png)
